onboard/catalog_actions.go

```diff
@@ -112,18 +114,18 @@ func (c *CatalogRepoActions) Init(ctx context.Context, baseCommit graveler.Commi
 }
 
 func (c *CatalogRepoActions) initBranch(ctx context.Context) error {
-	c.branchID = catalog.DefaultImportBranchName
-	branch, err := c.entryCataloger.GetBranch(ctx, c.repoID, catalog.DefaultImportBranchName)
+	c.branchID = DefaultImportBranchName
+	branch, err := c.entryCataloger.GetBranch(ctx, c.repoID, DefaultImportBranchName)
```

Comment:
When we get the branch and we fail, not with 'not found' we need to return the error - like we are doing when we try to create the branch.

Reply:
Right, done.